src/main/java/DukeManager.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+class DukeManager {
+    private static final int TODO = 5;
+    private static final int DEADLINE = 9;
+    private static final int BY = 3;
+    private static final int EVENT = 6;
+    private static final int FROM = 5;
+    private static final int TO = 3;
+    private final ArrayList<Task>taskList;
+
+    DukeManager(ArrayList<Task>tl) {
+        this.taskList = tl;
+    }
+
+    public void run() {
+        Scanner io = new Scanner(System.in);
+        String inputt = io.nextLine();
```

Comment:
Instead of using inputt maybe consider using input1/input2 or refer to the specific input.